tools/cockpit.spec

```diff
@@ -463,6 +463,14 @@ Provides: cockpit-sosreport = %{version}-%{release}
 Provides: cockpit-subscriptions = %{version}-%{release}
 Requires: subscription-manager >= 1.13
 %endif
+# NPM modules which are also available as packages
+Provides: bundled(js-jquery) = %{npm-version-jquery}
```

Comment:
I would prefer something like `%{npm-version:jquery}` to make it more obvious that `npm-version` is the keyword and `jquery` is the argument.

Reply:
If colons are legit in an RPM macro, then I agree. I actually thought of that, but wasn't sure. I'll try.

Reply:
Seems to work. Pushed, PTAL?